src/applications/gi/tests/e2e/03-vet-tec.e2e.spec.js

```diff
@@ -32,7 +31,7 @@ module.exports = E2eHelpers.createE2eTest(client => {
   GiHelpers.expectLocation(client, `/program-search`);
   GiHelpers.selectSearchResult(
     client,
-    UtilHelpers.createId(
+    GiHelpers.createId(
```

Comment:
This issue this is fixing is only in this branch

Reply:
It's addressing e2e failures.

Reply:
Once the container specific logic is moved out of helpers the issue will no longer occur. 
A prod flag is not needed for this story, as stated in assumption 1 this should be behind `gibctEstimateYourBenefits` which it will be once the container specific logic is moved to `src/applications/gi/containers/EstimateYourBenefits.jsx`

Reply:
This fix addresses all of the input fields in the Comparison Tool. I could scrap these changes, but we will just find ourselves doing the same work again down the road.

Reply:
- `src/applications/gi/components/Checkbox.jsx` needs prod flag
- `src/applications/gi/components/Dropdown.jsx` needs prod flag
- `src/applications/gi/components/RadioButtons.jsx` needs prod flag
- `src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx` is behind the feature flag
- `src/applications/gi/components/search/KeywordSearch.jsx` needs prod flag
- `src/applications/gi/components/vet-tec/VetTecEstimateYourBenefitsForm.jsx` is behind the feature flag
- `src/applications/gi/containers/EstimateYourBenefits.jsx` is behind the feature flag

What if you used the prod flag in the 4 components that need it to determine whether to use their current `handleFocus` vs the new one in helper util?
And then used new `handleInputFocus` from helpers in the 3 components that are behind the feature flag?